src/cli/config.ts

```diff
@@ -96,3 +98,21 @@ export function loadModuleContent<T>(relativePath: string): T | undefined {
 
   return integrationModule?.default ?? integrationModule;
 }
+
+async function isTypescriptPresent() {
+  const paths = await globby(path.join('src', '**', '*.ts'));
```

Comment:
This module is quite nice. I'll might replace the implementation of `walkDirectory` with this.